examples/fullscreen_plugin_example.py

```diff
@@ -4,6 +4,7 @@
 from folium.plugins import Fullscreen
 
 m = folium.Map(location=[41.9, -97.3], zoom_start=4)
-Fullscreen().add_to(m)
+Fullscreen(position = 'topright', title = 'Expand me', titleCancel = 'Exit me',
+              forceSeparateButton = True).add_to(m)
```

Comment:
We are enforcing `pep8`. That menas you will need to remove the spaces in the operators:

e.g.: `position = 'topright'` &rarr; `position='topright'`

and the indentation, like aligning the `forceSeparateButton` with the `position` above, note that this should be automatic in many text editors.